releases/release-1.24/README.md

```diff
@@ -34,60 +34,63 @@ description: |
 * [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
 * [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
 
-## TL;DR
+## Summary
 
-The 1.24 release cycle is proposed as follows:
+The 1.24 release cycle is as follows:
 
-- **Monday 10th January 2022**: Week 1 - Release cycle begins
-- **TBC**: Week 2 - [Production Readiness Soft Freeze](https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ)
-- **TBC**: Week 3 - [Enhancements Freeze](../release_phases.md#enhancements-freeze)
-- **TBC**: Week 11 - [Release Retrospective][Retrospective Document] part 1
-- **TBC**: Week 13 - [Code Freeze](../release_phases.md#code-freeze)
-- **TBC**: Week 14 - [Test Freeze](../release_phases.md#test-freeze)
-- **TBC**: Week 15 - Docs must be completed and reviewed
-- **Tuesday 19th April 2022**: Week 15 - Kubernetes v1.24.0 released
-- **TBC**: Week 16 - [Release Retrospective][Retrospective Document] part 2
-- **TBC**: Week 16 - [Release Retrospective][Retrospective Document] part 3
+- **Monday 10th January 2022**: Week 1 — Release cycle begins
+- **Thursday 27th January 2022**: Week 3 — [Production Readiness Soft Freeze](https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ)
+- **02:00 UTC Friday 4th February 2022** / **18:00 PST Thursday 3rd February 2022** : Week 4 — [Enhancements Freeze](../release_phases.md#enhancements-freeze)
+- **Wednesday 23rd March 2022**: Week 11 — [Release Retrospective][Retrospective Document] part 1
+- **01:00 UTC Wednesday 30th March 2022** / **02:00 BST Wednesday 30th March 2022** / **18:00 PDT Tuesday 29th March 2022**: Week 13 — [Code Freeze](../release_phases.md#code-freeze)
+- **01:00 UTC Wednesday 6th April 2022** / **02:00 BST Wednesday 6th April 2022** / **18:00 PDT Tuesday 5th April 2022**: Week 14 — [Test Freeze](../release_phases.md#test-freeze)
+- **Tuesday 12th April 2022**: Week 15 — Docs must be completed and reviewed
+- **Tuesday 19th April 2022**: Week 15 — Kubernetes v1.24.0 released
```

Comment:
Off by one? These are a week apart, I don't think they are both week 15?

Reply:
Good catch. Fixed in 7dddf3d24077118630aef9690198abdc585ed089.